src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/MessageListItem.vue

```diff
@@ -35,30 +36,39 @@ export default defineComponent({
       required: true,
     },
   },
-  emits: ["message-read"],
   setup() {
     const messageStore = useMessageStore();
+    const alertStore = useAlertStore();
 
-    return { messageStore };
+    return { messageStore, alertStore };
   },
   computed: {
     messageDate(): string {
-      return formatDate(String(this.message.notifiedOn), "LLL d, yyyy t");
+      return formatDate(String(this.message.latestMessageNotifiedOn), "LLL d, yyyy t");
     },
   },
 
   methods: {
     formatDate,
     async handleClick() {
       this.messageStore.currentMessage = this.message;
+      this.$props.message.isRead = true;
```

Comment:
I'm thinking using an $emit might be better for this case, see https://vuejs.org/guide/components/props.html#mutating-object-array-props

What do you think?

Reply:
@pcatkins - Agreed, fixed. Thanks!